pkg/datapath/linux/route/route_linux.go

```diff
@@ -514,6 +514,18 @@ func lookupDefaultRoute(family int) (netlink.Route, error) {
 	return routes[0], nil
 }
 
+func DeleteRouteTable(table, family int) error {
+	routes, err := netlink.RouteListFiltered(family, &netlink.Route{Table: table}, netlink.RT_FILTER_TABLE)
+	if err != nil {
+		return fmt.Errorf("Unable to list table %d routes: %s", table, err)
+	}
+
+	for _, route := range routes {
+		netlink.RouteDel(&route)
+	}
+	return nil
```

Comment:
This function is named `DeleteRouteTable`, but it appears to delete the rules *in* that table. I'm not familiar enough with the kernel API to say whether the table itself is a resource that needs to be deleted, but for readability would it be more accurate to call it something like `DeleteRoutesInTable(...)`? (or do we need an actual table delete call?)

Reply:
Looks like the kernel just adds and deletes tables as needed.

https://elixir.bootlin.com/linux/v4.19.182/source/net/ipv4/fib_frontend.c#L1424